csrankings-a.csv

```diff
@@ -1743,6 +1745,7 @@ Ann Q. Gates,University of Texas - El Paso,http://faculty.utep.edu/Default.aspx?
 Ann Quiroz Gates,University of Texas - El Paso,http://faculty.utep.edu/Default.aspx?alias=faculty.utep.edu/agates,NOSCHOLARPAGE
 Anna Bernasconi 0001,University of Pisa,http://www.di.unipi.it/~annab,rCnl9L4AAAAJ
 Anna Bou Ezzeddine,Kempelen Institute - KInIT,https://kinit.sk/member/anna-bou-ezzeddine/,aR9uQyUAAAAJ
+Anna Cinzia Squicciarini,Pennsylvania State University,https://ist.psu.edu/directory/acs20,wmleQ1QAAAAJ

```

Comment:
Not in EECS.

Reply:
Please see my previous comment. Anna works in cyber security.